modules/mapper-extras/src/main/java/org/elasticsearch/index/mapper/extras/RankFeatureFieldMapper.java

```diff
@@ -62,24 +64,53 @@ public static class Builder extends FieldMapper.Builder {
             true
         );
         private final Parameter<Map<String, String>> meta = Parameter.metaParam();
+        private final Parameter<Float> nullValue = new Parameter<>(
+            "null_value",
+            false,
+            () -> null,
+            (n, c, o) -> o == null ? null : objectToFloat(o),
+            m -> ft(m).nullValue,
+            XContentBuilder::field,
+            Objects::toString
+        ).addValidator(value -> {
+            if (value != null && value < Float.MIN_NORMAL) {
+                throw new IllegalArgumentException(
+                    "nullValue must be a positive normal float, for feature: [rank_feature] but it is "
+                        + value
```

Comment:
Let's get modify this error message to "[null_value] must be a positive normal float for field of type [rank_feature], got ".....